packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/deck/PapayaDeckControllerIntegrationTests.java

```diff
@@ -166,8 +166,8 @@ public void givenDeckRepresentationWhenWeCreateNewCardThenTheRepresentationDataA
 
     @Test
     @DisplayName("Given a deck representation, when we delete an existing card, then the representation data are updated")
-    @Sql(scripts = {"/scripts/papaya.sql"}, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
-    @Sql(scripts = {"/scripts/cleanup.sql"}, executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, config = @SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED))
+    @Sql(scripts = { "/scripts/papaya.sql" }, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
```

Comment:
There's no need to change anything here, you would end up changing tons of files with your formatter setting I imagine